inc/functions_post.php

```diff
@@ -1019,7 +1019,20 @@ function get_post_attachments($id, &$post)
 					}
 					elseif((($attachment['thumbnail'] == "SMALL" && $forumpermissions['candlattachments'] == 1) || $mybb->settings['attachthumbnails'] == "no") && $isimage)
 					{
-						eval("\$post['imagelist'] .= \"".$templates->get("postbit_attachments_images_image")."\";");
+						if ($forumpermissions['candlattachments'])
+						{
+							eval("\$post['imagelist'] .= \"".$templates->get("postbit_attachments_images_image")."\";");
+						} 
+						else 
+						{
+							eval("\$post['thumblist'] .= \"".$templates->get("postbit_attachments_thumbnails_thumbnail")."\";");
+							if($tcount == 5)
+							{
+								$thumblist .= "<br />";
```

Comment:
I can't seem to work out where this `$thumblist` variable actually gets used (yes it's used a couple of lines above too), it doesn't seem to be used in a template or anything and is only ever written to and not read. I think it's supposed to be:

```suggestion
								$post['thumblist'] .= "<br />";
```

Any thoughts @mybb/developers @mybb/designers? This shouldn't hold this PR back, but it also affects here: https://github.com/mybb/mybb/blob/ad721421d55f3feac633a44f89c151c380c0c007/inc/functions_post.php#L1015